File: php/inmobi/monetization/ads/Interstitial.php
<?php

require_once(dirname(__FILE__)."/Banner.php");
/**
 * Publishers may use this class to request for Interstitial ads.
 * 
 * @note Interstitials are full screen ads.
 * @author rishabhchowdhary
 * 
 */
class Interstitial extends Banner {

	function __construct() {
		parent::__construct();
		$this->requestType = AdRequest::INTERSTITIAL;
     }

}

?>

File: android/android_inmobi/src/main/java/com/inmobi/monetization/ads/Interstitial.java
package main.java.com.inmobi.monetization.ads;

import main.java.com.inmobi.monetization.api.request.enums.AdRequest;

/**
 * Publishers may use this class to request for Interstitial ads.
 * 
 * @note Interstitials are full screen ads.
 * @author rishabhchowdhary
 * 
 */
public class Interstitial extends Banner {


	public Interstitial() {
		requestType = AdRequest.INTERSTITIAL;
	}
}
